build/legacy-mongo-shell/test.example.js

```diff
@@ -0,0 +1,29 @@
+// Move to test.js and run with `task mongo-test`.
+
+(function() {
+    "use strict";
+
+    const col = db.test;
```

Comment:
rename col to `coll`.

Reply:
Why? FWIW, in FerretDB code base, there is one instance of `col` and zero instances of `coll`

Reply:
I find `coll` more readable. I am also basing this off of MongoDB's style.

Reply:
Is it public?

Reply:
I meant their jstest script files.

Reply:
Is their code style is written down and public?

Reply:
> Is their code style is written down and public?

Yes. [Here](https://github.com/mongodb/mongo/wiki/Server-Code-Style).

> For anything that isn't explicitly covered here, default to the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html) and the [Google JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml).